### ISSUE TITLE
Data labeling

### ISSUE THREAD

**Issue by Json-To-String** [ssc40/MLharvest#11]
We've settled on the vegetation index (NDVI) on being our output metric (the one we want our model to predict) from an input image. 

Let's find the vegetation indices for our training data.

**Comment by Json-To-String** [comment]
Ideas: 

$f(x)$, where $x$ could either be:

- Date, and $f(x)$ is the predicted image of what the area should look like then
- An image and a look ahead parameter, and $f(x)$ is the predicted image of the area that long ahead. For simplicity or a first pass, let's keep the look ahead set to 6 months. I suppose that changes things to $f(x,y)$.

This will mean we need:

- [ ] to get a date tag for each image
- [ ] to restructure the inputs and outputs of the neural network we've been working with to consider the size and shape of the images (since they won't be the same as the CIFAR dataset) (this is just #10 I realize)

**Comment by ssc40** [comment]
<img width="1470" height="1445" alt="Image" src="https://github.com/user-attachments/assets/99581c55-98b7-484b-9797-f30d84e9c57d" />

This is an example of the watermarked image

**Comment by Json-To-String** [comment]
Potentially useful:
https://dataspace.copernicus.eu/analyse/apis/catalogue-apis

Mentions:
- OData
- STAC
- OpenSearch
- Sentinel Hub Catalog API

**Comment by Json-To-String** [comment]
TODO:


https://dataspace.copernicus.eu/analyse/apis/catalogue-apis

Let's find which of these is easy to get an image with:
- [ ] Geometry - Ideally its the region we specify in `data/KML/hirapur.kml`, but we could probably settle for a square and then maybe crop it later. (It'll probably be a bitwise mask like we did for the clouds). 
- [x] Datetime

Go through each of these and rank these in ease of use and whether or not they'll work.
Good things to try is to see if they have example notebooks in python, or in their API reference they seem to give you the above. You may have to do a little github reading, but you don't have to actually *get* the image, unless its painfully easy. 
- OData
- STAC
- OpenSearch
- Sentinel Hub Catalog API

**Comment by ssc40** [comment]
Here is the center of the hirapur region by coordinate: 

<img width="152" height="20" alt="Image" src="https://github.com/user-attachments/assets/4175fba4-4404-4a85-802e-97920d20d791" />

**Comment by ssc40** [comment]
ee engine has the ability to find the date through image id, as well as lots of other useful metadata in properties

**Comment by Json-To-String** [comment]
Closed with #22
